css/evaluations.css

```diff
@@ -0,0 +1,110 @@
+body{
+	background-color: white;
+}
+
+.logoClass{
+	background-image: url('https://www.softvision.ro/images/common/logo-v2.png');
+	width: 200px;
+	height: 37.40px;
```

Comment:
Because I was looking at the original logo dimensions found on internet. After that, to keep things simple, I said I want the logo to be 200px width, so I had to calculate the hight of the logo, so the proportion is the same.